node/src/bootstrapper.rs

```diff
@@ -544,51 +562,6 @@ impl Bootstrapper {
         }
     }
 
-    #[cfg(test)] // The real ones are private, but ActorSystemFactory needs to use them for testing
```

Comment:
No more static `CryptDE`s, so tests for the functions to manipulate them are obsolete.